client/js/components/map/map/DpOlMapEditFeature.vue

```diff
@@ -56,7 +56,11 @@
       type="button"
       @click="toggle"
       data-cy="editButtonDesc"
-      v-tooltip="Translator.trans('explanation.territory.help.edit',{ editTool: Translator.trans('map.territory.tools.edit') })"
+      v-tooltip="{
+        classes: this.tooltipClass,
+        container: this.$parent.$parent.$parent.$el,
```

Comment:
You cant know that, but in preparation of the migration to vue3 we have to find other solutions than `.$el` since with vue3 this doesn't work anymore because then there can be more root elements than one. So the explanation here, and my experience. Using `refs` looks like reasonable alternative
https://stackoverflow.com/questions/61600078/vue-3-composition-api-how-to-get-the-component-element-el-on-which-the-comp
even though its explained different here:
https://vuejs.org/api/component-instance.html#el

Reply:
thanks for the hint. i replaced `this.$el` with `this.$refs.<ref>` where appropriate and was even able to remove some container options entirely in 1961b3f1e64c761d3fcae29240889bed1aea54e1